package.json

```diff
@@ -5,6 +5,7 @@
   "main": "server/app.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "node node_modules/json-schema-to-typescript/dist/cli.js server/config/questions.schema.json && tsc",
```

Comment:
you could install `typescript` under `devDependencies` and then call `node_modules/.bin/tsc`
then we can skip the whole install typescript step.